src/microbe_stage/Microbe.cs

```diff
@@ -1427,22 +1427,39 @@ private void OnIGotAddedToColony()
             return;
 
         var oldRotation = Rotation;
-        var vectorToParent = GlobalTransform.origin - ColonyParent.GlobalTransform.origin;
-        ChangeNodeParent(ColonyParent);
 
-        var vectorToParentRotated = vectorToParent.Rotated(Vector3.Down, Rotation.y);
-        var vectorToMembrane = Membrane.GetExternalOrganelle(vectorToParentRotated.x, vectorToParentRotated.y);
+        // Gets the global rotation of the parent
+        var globalParentRotation = ColonyParent.GlobalTransform.basis.GetEuler();
+
+        // A vector from the parent to me
+        var vectorFromParent = GlobalTransform.origin - ColonyParent.GlobalTransform.origin;
+
+        // This vector gets as if I had no rotation.
+        // This is important, because GetExternalOrganelle only works with non-rotated microbes
+        var vectorToParentRotated = (-vectorFromParent).Rotated(Vector3.Down, Rotation.y);
```

Comment:
`vectorFromParent` is simply the vector from the parents' center towards mine.
`-vectorFromParent` is the vector from my center towards the parents'

the problem is that `GetExternalOrganelle` does not take the microbe rotation in effect, which means I have to do it. `vectorToParentRotated` is the vector from my center towards the parent's if I had no rotation (pointing upwards).

Reply:
So, this is what I understood. But I find the name `vectorToParentRotated` misleading, because it is the vector to parent *when not rotated*. I think that speaking of referential or internal representation may be more enlightening over what is going on here.